src/common/structure/block.go

```diff
@@ -187,3 +188,7 @@ func (b *Block) GetTraceID() string {
 func (b *Block) GetTagsAttributeName() string {
 	return b.TagsAttributeName
 }
+
+func (b *Block) IsGCPBlock() bool {
+	return false
```

Comment:
@nimrodkor Let's extend the field to indicate the cloud provider from a closed set.. don't you think?